beeline/patch/test_urllib.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+import six
+import unittest
+from mock import Mock, patch
+
+import urllib
+
+class TestUrllibPatch(unittest.TestCase):
+    @unittest.skipIf(six.PY2, "urllib not compatible with python2")
+    def test_request_fn_injects_headers_and_returns(self):
+        from beeline.patch.urllib import _urllibopen # pylint: disable=bad-option-value,import-outside-toplevel
```

Comment:
Is this imported here because it's not compatible with python2?

Reply:
yes, otherwise when the files are imported it'll fail